### PR TITLE
Allow admin user to delete an existing article

### PR DESCRIPTION
**Pull request by markoparfeniuk**
Add article action menu

### CODE REVIEW

**Review by Yurii-Hubych — APPROVED** [review]
LGTM

File: src/components/Toasts/FailureToast/FailureToast.js [hunk 1]
```diff
@@ -24,11 +24,12 @@ const dropIn = {
 
 export default function FailureToast({ t, handleRetry }) {
     return (
```
Comment (Juli11V) [inline src/components/Toasts/FailureToast/FailureToast.js:26]: Please add extra line;)

File: src/components/Toasts/SuccessToast/SuccessToast.js [hunk 2]
```diff
@@ -23,11 +23,12 @@ const dropIn = {
 
 export default function SuccessToast({ t, message, details }) {
     return (
```
Comment (Juli11V) [inline src/components/Toasts/SuccessToast/SuccessToast.js:25]: The same

File: src/services/Articles/ArticleService.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                "Content-Type": "application/json",
+                "Access-Control-Allow-Origin": "*",
+                Authorization: `Bearer ${this.token}`,
+            },
+            method: method,
+        }
+        if (data) {
+            options.body = JSON.stringify(data);
+        }
+        return fetch(url, options);
```
Comment (Juli11V) [inline src/services/Articles/ArticleService.js:22]: The same

File: src/services/Articles/ArticleService.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    get(id) {
+        const method = 'GET';
+        let url = urls.articlesEndpoint;
+        if (id) {
+            url = `${url}/${id}`;
+        }
+        return this.request(url, method).then(res => res.json());
```
Comment (Juli11V) [inline src/services/Articles/ArticleService.js:32]: Here

File: src/services/Articles/ArticleService.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    delete(id) {
+        const method = 'DELETE';
+        let url = urls.articlesEndpoint;
+        if (id) {
+            url = `${url}/${id}`;
+        }
+        return this.request(url, method).then(res => res.ok ? res.json() : Promise.reject(res.status));
```
Comment (Juli11V) [inline src/services/Articles/ArticleService.js:42]: And here

File: src/components/Backdrop/Backdrop.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import { motion } from "framer-motion"
+import "./Backdrop.scss"
+
+export default function Backdrop({children, onClick}) {
+    return (
```
Comment (Juli11V) [inline src/components/Backdrop/Backdrop.js:5]: Add line

**Review by Juli11V — APPROVED** [review]
Good work ;)

**Pull request merged**